test/imports/mixins.css

```diff
@@ -0,0 +1,8 @@
+@import 'imports/mixins';
```

Comment:
I think you mean to import this in your test and not in the file itself.

Reply:
d'oh!